sklearn/model_selection/_search.py

```diff
@@ -234,8 +241,8 @@ def __iter__(self):
             if grid_size < self.n_iter:
                 raise ValueError(
                     "The total space of parameters %d is smaller "
-                    "than n_iter=%d." % (grid_size, self.n_iter)
-                    + " For exhaustive searches, use GridSearchCV.")
+                    "than n_iter=%d." % (grid_size, self.n_iter) +
+                    " For exhaustive searches, use GridSearchCV.")
```

Comment:
Quite right, this is the dread hand of AutoPEP8. Apologies, I'll switch it
off in future.

On Tue, 1 Mar 2016, 14:54 Raghav R V, <notifications@github.com> wrote:

> In sklearn/model_selection/_search.py
> <https://github.com/scikit-learn/scikit-learn/pull/6160#discussion_r54568636>
> :
>
> > @@ -234,8 +241,8 @@ def __iter__(self):
> >              if grid_size < self.n_iter:
> >                  raise ValueError(
> >                      "The total space of parameters %d is smaller "
> > -                    "than n_iter=%d." % (grid_size, self.n_iter)
> > -                    + " For exhaustive searches, use GridSearchCV.")
> > +                    "than n_iter=%d." % (grid_size, self.n_iter) +
> > +                    " For exhaustive searches, use GridSearchCV.")
>
> Could you please avoid changing lines that are not part of your PR?
> (That will help speed up the review of the actual content of your PR,
> helping you to merge it sooner into scikit-learn :) )
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scikit-learn/scikit-learn/pull/6160/files#r54568636>.
>